src/Adapters/Files/AzureBlobStorageAdapter.js

```diff
@@ -0,0 +1,116 @@
+// AzureBlobStorageAdapter
+//
+// Stores Parse files in Azure Blob Storage.
+
+import * as azure from 'azure-storage';
+import { FilesAdapter } from './FilesAdapter';
+
+export class AzureBlobStorageAdapter extends FilesAdapter {
+    // Creates an Azure Storage client.
+    // Provide storage account name or storage account connection string as first parameter
+    // Provide container name as second parameter
+    // If you had provided storage account name, then also provide storage access key
+    // Host is optional, Azure will default to the default host
+    // directAccess defaults to false. If set to true, the file URL will be the actual blob URL
+    constructor(
```

Comment:
would be more consistent with other adapters to only have one parameter `options`